e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/your-buyer/check-your-answers-your-buyer-page.spec.js

```diff
@@ -0,0 +1,115 @@
+import {
+  headingCaption,
+  submitButton,
+  saveAndBackButton,
+} from '../../../../pages/shared';
+import partials from '../../../../partials';
+import {
+  BUTTONS,
+  PAGES,
+} from '../../../../../../content-strings';
+import { ROUTES } from '../../../../../../constants';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+
+const {
+  ROOT,
+  START,
+  ALL_SECTIONS,
+  CHECK_YOUR_ANSWERS: {
+    YOUR_BUSINESS,
+    YOUR_BUYER,
+  },
+} = ROUTES.INSURANCE;
+
+const CONTENT_STRINGS = PAGES.INSURANCE.CHECK_YOUR_ANSWERS.YOUR_BUYER;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswersAndSubmit;
+
+context('Insurance - Check your answers - Your buyer - I want to confirm my selection for the your buyer section of my export insurance application', () => {
```

Comment:
Can we add 'page' so it's "Your buyer page" just to be consistent? Same for all check your answer test descriptions